public/keymaps/a/ada_ada1800mini_default.json

```diff
@@ -1,22 +1,22 @@
 {
   "keyboard": "ada/ada1800mini",
   "keymap": "default",
-  "commit": "3057e5f8ca523427644ca1e8745bc0b5492a6d67",
+  "commit": "b7771ec25b96f2b88a7fa4201081e10ca6fbb9d4",
   "layout": "LAYOUT",
   "layers": [
     [
-      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_BSPC",    "KC_MINS", "KC_EQL",  "KC_NUM",
+      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_BSPC",    "KC_MINS", "KC_EQL",  "KC_NLCK",
```

Comment:
:(

Reply:
Didn't want to throw the branch away after I spent over a week working on it.

I'll fix these soonish.